src/elm/interval.coffee

```diff
@@ -2,7 +2,7 @@
 { ThreeValuedLogic } = require '../datatypes/logic'
 { build } = require './builder'
 { Quantity, doAddition, doSubtraction, compare_units, convert_value } = require '../datatypes/quantity'
-{ successor, predecessor } = require '../util/math'
+{ successor, predecessor, maxValueForInstance, MAX_DATETIME_VALUE, minValueForInstance, MIN_DATETIME_VALUE } = require '../util/math'
```

Comment:
Are you using `maxValueForInstance` or `minValueForInstance`?  I don't see any usages in the diff.

Reply:
Good catch. These have been removed.